conf/test_alternatives.config

```diff
@@ -40,15 +40,16 @@ params {
     build_krakenuniq            = true
     krakenuniq_keepintermediate = true
 
-    krakenuniq_build_options     = "--work-on-disk --max-db-size 14 --kmer-len 15 --minimizer-len 13 --jellyfish-bin \"\$(which jellyfish)\""
-    malt_build_options           = "--sequenceType Protein"
+    krakenuniq_build_options    = "--work-on-disk --max-db-size 14 --kmer-len 15 --minimizer-len 13 --jellyfish-bin \"\$(which jellyfish)\""
+    malt_build_options          = "--sequenceType Protein"
 
     accession2taxid             = params.pipelines_testdata_base_path + 'createtaxdb/data/taxonomy/nucl_gb.accession2taxid'
     nucl2taxid                  = params.pipelines_testdata_base_path + 'createtaxdb/data/taxonomy/nucl2tax.map'
     prot2taxid                  = params.pipelines_testdata_base_path + 'createtaxdb/data/taxonomy/prot.accession2taxid.gz'
     nodesdmp                    = params.pipelines_testdata_base_path + 'createtaxdb/data/taxonomy/nodes.dmp'
     namesdmp                    = params.pipelines_testdata_base_path + 'createtaxdb/data/taxonomy/names.dmp'
     malt_mapdb                  = 's3://ngi-igenomes/test-data/createtaxdb/taxonomy/megan-nucl-Feb2022.db.zip'
```

Comment:
Could it be that the test's are failing because of the filename containing `'-'` ?

Reply:
It was that the folder name had a `.db` suffix :sweat_smile: